test/integration/gobierto_admin/gobierto_plans/projects/project_versions_test.rb

```diff
@@ -24,6 +24,10 @@ def site
           @site ||= sites(:madrid)
         end
 
+        def default_test_context
+          { site: site, js: true, admin: admin, window_size: :xl }
```

Comment:
@ferblape @entantoencuanto if you happen to have a tests which fails due to some page contents being invisible due to window size, you can resize it like so.